llvm/include/llvm/IR/IntrinsicsAMDGPU.td

```diff
@@ -2783,7 +2783,8 @@ class AMDGPUGlobalLoadLDS :
      llvm_i32_ty,                       // imm offset (applied to both global and LDS address)
      llvm_i32_ty],                      // auxiliary data (imm, cachepolicy (bit 0 = sc0,
                                         //                                   bit 1 = sc1,
-                                        //                                   bit 4 = scc))
+                                        //                                   bit 4 = scc,
+                                        //                                   bit 31 = volatile (compiler implemented)))
```

Comment:
We ought to get this in the AMDGPUUsage documentation for intrinsics (and we really should get tablegenerated docs for this)

Reply:
Yeah ... should making a table of `aux` bits bi this PR?